index.html

```diff
@@ -12,6 +12,52 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header>
+        <img src="img/karma-logo.svg" alt="karma logo" class="logo">
+            <ul class="navbar">
+                <li><a href="#">Meet Karma </a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>   
+    </header>
+
+    <main>
+        <section class="main">
+           <h1>Introducing Karma</h1>
+           <h2>Bring WIFI with you, everywhere you go</h2>  
+                <div class="button">
+                    <p>Learn More</p>
+                </div>    
+        </section>
+        <p class ="tagline">Everyone needs a little Karma</p>
+        <container class="mobile-view">
```

Comment:
I'm afraid the container tag doesn't exist! We speak of containers as a concept, but there is no specific tag for them. You can use a `section` here, and give it a class of `container` instead. 
You can also probably move the 'tagline' inside of it, and make it a heading tag instead of a paragraph.